index.html

```diff
@@ -15,6 +15,24 @@
                 padding: 20px 0;
                 border-bottom: 1px solid #eee;
             }
+            ul {
+                margin: 0;
+                padding: 0;
+            }
+            ul li {
+                cursor: pointer;
```

Comment:
Change the cursor to hand instead of pointer.
Make the list items unselectable for a cleaner UX. 
Use `user-select: none`